src/main/resources/com/chikli/hudson/plugin/naginator/Messages.properties

```diff
@@ -1,2 +1,9 @@
 NaginatorCause.Description=Started by Naginator after build {0} failure
+RegexpForMatrixStrategy.TestChildrenRetriggerAll=Test failed matrix parts, and retrigger all
+RegexpForMatrixStrategy.TestChildrenRetriggerMatched=Test failed matrix parts, and retrigger only matched parts
+RegexpForMatrixStrategy.TestParent=Test the parent build
+NoChildStrategy.RerunWhole.DisplayName=Rerun the matrix with all combinations
+NoChildStrategy.RerunEmpty.DisplayName=Rerun the matrix with no combinations
+NoChildStrategy.DontRerun.DisplayName=Don''t rerun
```

Comment:
They are MessageFormat strings and single quotes are treated as escaping characters.
You have to write `''` to represent `'`.

Please refer the document for `MessageFormat` for more details: http://docs.oracle.com/javase/6/docs/api/java/text/MessageFormat.html